.circleci/config.yml

```
@@ -0,0 +1,31 @@
+version: 2.1
+
+defaults: &defaults
+  working_directory: ~/placement-prototypes
+
+orbs:
+  aws-s3: circleci/aws-s3@3.0
+
+jobs:
+  upload:
+    <<: *defaults
+    docker:
+      - image: 'cimg/python:3.6'
+    steps:
+      - checkout
+      - aws-s3/sync:
+          arguments: |
+            --exclude \"*.DS_Store*\"
+            --cache-control "public,max-age=604800"
```

I would cache it for just 10 mins because we don't have another cache invalidation strategy.

```suggestion
            --cache-control "public,max-age=600"
```

---

ok